web/src/dialog_widget.ts

```diff
@@ -89,14 +89,16 @@ export function hide_dialog_spinner(): void {
     loading.destroy_indicator($spinner);
 }
 
-export function show_dialog_spinner(): void {
```

Comment:
This is re-added in the next commit. There is no need to remove this here in the first place.

Reply:
@sahil839 I have fix that you can review now.